firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1832,26 +1832,21 @@ extension String {
                 tableName: "Settings",
                 value: "Default Search Engine",
                 comment: "Title for the `default search engine` settings section in the Search page in the Settings menu.")
-            public static let DefaultSearchEngineFooter = MZLocalizedString(
-                key: "Settings.Search.DefaultSearchEngine.Footer.v122",
-                tableName: "Settings",
-                value: "Results from searches, history, bookmarks, and more",
-                comment: "Footer for for the `default search engine` settings section in the Search Settings page, which explains in more details what the `Show Search Suggestions` setting includes.")
-            public static let QuickSearchEnginesTitle = MZLocalizedString(
-                key: "Settings.Search.QuickEnginesTitle.v121",
+            public static let AlternateSearchEnginesTitle = MZLocalizedString(
+                key: "Settings.Search.AlternateSearchEngines.Title.v124",
                 tableName: "Settings",
-                value: "Quick-Search Engines",
-                comment: "Title for quick-search engines settings section in the Search page in the Settings menu.")
-            public static let PrivateSessionTitle = MZLocalizedString(
-                key: "Settings.Search.PrivateSession.Title.v122",
+                value: "Alternate Search Engines",
```

Comment:
I don't think this is correct in English.
https://www.merriam-webster.com/dictionary/alternate

Pretty sure this should be alternative, but there are no specs attached anywhere.

Reply:
"Alternate" works well here, imo; it's often read as "other" as per definition 3. "alternative" also works, too.

Reply:
Do we have specs for this UI?

Reply:
[Link To Figma](https://www.figma.com/file/Ha2T4I0nX0h7B95GYxH6wV/Simplified-UI-for-Private-Mode?type=design&node-id=7606-16267&mode=design&t=G7N7qtwstc5Bh1mQ-0)

Reply:
Could you point me to the full specs (on Figma I suppose)?

@adudenamedruby 
Not sure which one is definition 3. The third definition (`3 of 3`) is for a noun, the 3rd bullet point in the 1st (`1 of 3`) definition is "every other day".

With that said, I expect locales to get it wrong (pt-BR already did).

CC @Delphine

Reply:
In that case, this should be raised with copy. Summoning @ewachowiak

Reply:
Please work with Tina Vaamonde who is the content designer for this project.